test/integration/scheduler/plugins/plugins_test.go

```diff
@@ -2677,7 +2677,7 @@ func TestSchedulingGatesPluginEventsToRegister(t *testing.T) {
 		{
 			name:          "preEnqueue plugin with event registered",
 			enqueuePlugin: &SchedulingGatesPluginWithEvents{SchedulingGates: schedulinggates.SchedulingGates{}},
-			count:         3,
+			count:         2,
```

Comment:
@kerthcet you added this test. Do you think it still makes sense to keep?
Or should we be testing it a different way?

Reply:
sounds good, let's leave it for a follow up.